### PR TITLE
refactor: 💡 Create interface for getGroups method

### PR DESCRIPTION
**Pull request by polymath-eric**

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -939,6 +939,14 @@ export interface Permissions {
  */
 export type GroupPermissions = Pick<Permissions, 'transactions' | 'transactionGroups'>;
 
+/**
+ * This represents all Permission Groups of a Security Token
+ */
+export interface AllPermissionGroups {
```
Comment (monitz87) [inline src/types/index.ts:945]: What about

```
/**
 * This represents all Permission Groups of a specific Security Token, separated by category
 */
export interface CategorizedPermissionGroups {
```

**Review by monitz87 — CHANGES_REQUESTED** [review]
Minor naming suggestion. Other than that, all good, and good catch on the spelling mistakes

File: src/types/index.ts (hunk 1 above)
Comment (polymath-eric) [reply src/types/index.ts:945]: What about just `PermissionGroups`, it looks like that won't conflict with anything.

I think Categorized is better than All, but `known` and `custom` feel like pretty broad categories to me.

Comment (polymath-eric) [reply src/types/index.ts]: Pushed a commit with Categorized, but I can change it to just `PermissionGroups` also. 

Let me know what you prefer.

Comment (monitz87) [reply src/types/index.ts]: `PermissionGroups` is better I think

Comment (polymath-eric) [reply src/types/index.ts]: Updated to PermissionGroups

**Review by monitz87 — APPROVED** [review]

**Pull request merged**